src/backend/src/common/constants/pia-types.constant.ts

```diff
@@ -2,19 +2,15 @@ import { PiaTypesEnum } from '../enums/pia-types.enum';
 
 export const PiaTypes = [
```

Comment:
Are we addressing PIA types based on index? Can we wrap them as an object for easier/readable de-refrencing?

Reply:
@antsand  Are you suggesting something like 👇 
```
{
 [PiaTypesEnum.STANDARD]: 'Standard PIA',
  ...
}
```

Reply:
Also, doesn't NestJS have support for an actual enum type?

Reply:
What sort of NestJS support are you referring to @Bottle7? enum type support is a TypeScript feature, if any. Refer any `.enum.ts` files in the backend codebase for more references. Please let me know if that did not answer your question.

Reply:
Sure. There IS an enum type and we can leverage it to structure the data like @antsand is suggesting.

Reply:
hmm, not quite:

"STANDARD": {
     "value": "Standard PIA"
}

this can still be referenced in JS as Object.STANDARD.value